_posts/2019-11-17-Frostgram.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Frostgram Highrate Startchar
+website: Frostgram.eu
+author: Slife
+image: https://share-your-photo.com/img/4e97e9a5fe.jpg
+tags: wotlk  pvp highrate Startchar
+published: true
+---
+
+Why we took azeroth core? Because after test it is one of the bug free cores that is currently available and very well!
```

Comment:
`Why we took azeroth core? Because... ` should be reworded in `We took AzerothCore because...`

also I wouldn't say "bug free" as there is no software without bugs. The entire sentence could be rewritten as:

`We choese AzerothCore because after testing it, we found out that it is one of the cores with less bug that is currently available!`

if the original author agrees with this change

Reply:
done